README.md

```diff
@@ -19,32 +19,39 @@
 [![Documentation License](https://img.shields.io/badge/doc%20license-CC--BY%204.0-green)](LICENSE.md)
 [![Documentation - Read Now!](https://img.shields.io/badge/doc-read%20now%20%E2%9E%94-blueviolet)](https://vhdl.github.io/pyVHDLModel/)
 
-# pyVHDLModel
```

Comment:
Why did you remove the headline?
Or is you logo broken?

![image](https://user-images.githubusercontent.com/956109/134823141-8394f80a-89da-46b7-a052-656214adf32a.png)